packages/traceability-schemas/src/generators/CommodityType.js

```diff
@@ -0,0 +1,14 @@
+const faker = require('faker');
+
+const getCommodityType = () => {
```

Comment:
This looks great, but I wonder if we ought to start using static JSON fixtures?

Reply:
Oh right, not even using it. Copy-paste oversight.